01week/rockPaperScissors.js

```diff
@@ -9,7 +9,25 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
-
+let h1 = hand1.toLowerCase().trim();
+//console.log("h1 is "+h1);
+let h2 = hand2.toLowerCase().trim();
+//console.log("h2 is "+h2);
+if ((h1===h2)&&
+((h1==="paper")||(h1==="scissors")||(h1==="rock"))&&
```

Comment:
I would done the validation at the very beginning of the function instead of mixing it in with the "Tie" check.